src/components/header/Header.js

```diff
@@ -1,29 +1,48 @@
 import React from "react";
 import { Col, Input, Row } from "antd";
 import Logo from "./Logo";
+import lonceng from "../../assets/img/lonceng.png";
+import product from "../../assets/img/product.png";
```

Comment:
The name `product` is too generic. Please rename it. If you confused about the UI component from Figma, you can ask UI/UX team.